services.yaml

```diff
@@ -3172,7 +3140,29 @@ definitions:
       initiator_type:
         type: string
         example: "User"
-  ServiceBasic:
+  EnvironmentWithRole:
+    type: object
+    description: Extended information about a environment key
```

Comment:
an environment

Reply:
fixed to: Environment data including user's role